Neptune.Database/dbo/Views/dbo.vTreatmentBMPAssessmentDetailed.sql

```diff
@@ -77,6 +78,7 @@ join dbo.FieldVisitType fvt on fv.FieldVisitTypeID = fvt.FieldVisitTypeID
 join dbo.Person fvp on fv.PerformedByPersonID = fvp.PersonID
 join dbo.StormwaterJurisdiction sj on bmp.StormwaterJurisdictionID = sj.StormwaterJurisdictionID
 join dbo.Organization o on sj.OrganizationID = o.OrganizationID
+join dbo.WaterQualityManagementPlan wqmp on bmp.WaterQualityManagementPlanID = wqmp.WaterQualityManagementPlanID
 --left join tbopf tbo20 on tba.TreatmentBMPAssessmentID = tbo20.TreatmentBMPAssessmentID and tbo20.TreatmentBMPAssessmentObservationTypeID = 20

```

Comment:
you cannot use an inner join here!  it needs to be a left join, since a bmp can belong to 0 or 1 wqmp.